openedx/features/qverse_features/registration/validators.py

```diff
@@ -23,21 +23,28 @@ def validate_admission_file(file):
                 'regno', 'firstname', 'surname', 'othername', 'levelid',
                 'programmeid', 'departmentid', 'mobile', 'email'
                 ]
-    header_row = []
+
     try:
         file_content = file.read()
         
         try:
-            header_row = get_file_header_row(file_content, 'utf-8')
+            file_rows = get_file_rows(file_content, 'utf-8')
         except Error:
-            header_row = get_file_header_row(file_content, 'utf-16')
-
+            file_rows = get_file_rows(file_content, 'utf-16')
+    except TypeError:
+        raise ValidationError('Invalid file content. Csv should only use "," as delimiter')
     except Exception:
         raise ValidationError('Invalid file encoding format. Only utf-8 and utf-16 file encoding formats are supported.')
```

Comment:
I think this should be handled in the inner `try` block, where we are actually trying to read the file.

Reply:
@tasawernawaz I've handled it in outer try/except because this exception is expected from lines (utf-8, utf-16).